moped-editor/src/queries/dashboard.js

```diff
@@ -32,7 +32,7 @@ export const USER_PERSONNEL_PROJECTS_QUERY = gql`
           project_name
           current_phase
           current_status
-          moped_proj_notes(where: {project_note_type: {_eq: 2}}, order_by: {date_created: desc}) {
+          moped_proj_notes(where: {project_note_type: {_eq: 2}, status_id: {_eq:1}}, order_by: {date_created: desc}) {
```

Comment:
@mddilley heads up mike, I had to include this line to prevent deleted statuses from showing up on the dashboard.